src/components/InternalTransfer/index.tsx

```diff
@@ -0,0 +1,179 @@
+import classnames from 'classnames';
+import React, { useCallback, useMemo, useState } from 'react';
+import { Button } from 'react-bootstrap';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { DropdownComponent } from '../';
+import { isNicknamesEnabled } from '../../api';
+import { CloseIcon } from '../../assets/images/CloseIcon';
+import { Modal } from '../../components';
+import { useWalletsFetch } from '../../hooks';
+import { createInternalTransfersFetch, selectUserInfo, selectWallets } from '../../modules';
+import { InternalTransferInput } from './InternalInput';
+
+export const InternalTransferComponent = () => {
+    const { formatMessage } = useIntl();
+
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    const wallets = useSelector(selectWallets);
+    const user = useSelector(selectUserInfo);
+
+    const [username, setusername] = useState('');
+    const [currency, setCurrency] = useState('');
+    const [amount, setAmount] = useState('');
+    const [otp, setOtp] = useState('');
+
+    const [show, setShow] = useState(false);
+
+    useWalletsFetch();
+
+    const walletsList = wallets.length ? wallets.map(item => item.currency && item.currency.toUpperCase()) : [];
+
+    const wallet = wallets.length ? wallets.find(item => item.currency.toLowerCase() === currency.toLowerCase()) : undefined;
```

Comment:
```suggestion
    const wallet = wallets.length > 0 && wallets.find(item => item.currency.toLowerCase() === currency.toLowerCase());
```
I think this would be enough